drivers/hid/i2c-hid/i2c-hid-core.c

```diff
@@ -182,8 +182,8 @@ static const struct i2c_hid_quirks {
 		I2C_HID_QUIRK_NO_RUNTIME_PM },
 	{ I2C_VENDOR_ID_GOODIX, I2C_DEVICE_ID_GOODIX_01F0,
 		I2C_HID_QUIRK_NO_RUNTIME_PM },
-	{ USB_VENDOR_ID_ELAN, HID_ANY_ID,
-		 I2C_HID_QUIRK_BOGUS_IRQ },
+	{ USB_VENDOR_ID_ELAN, I2C_PRODUCT_ID_ELAN_TOUCHPAD,
```

Comment:
It should use `I2C_VENDOR_ID_ELAN`, but since there is an example of `USB_VENDOR_ID_ELAN` in use, it's okay for now.